packages/commands/src/components/command-menu.js

```diff
@@ -62,7 +63,7 @@ function CommandMenuLoader( { name, search, hook, setLoader, close } ) {
 						{ command.icon && <Icon icon={ command.icon } /> }
 						<span>
 							<TextHighlight
-								text={ command.label }
+								text={ decodeEntities( command.label ) }
```

Comment:
Thanks for the review. Can we call it [here](https://github.com/WordPress/gutenberg/blob/trunk/packages/core-commands/src/site-editor-navigation-commands.js#L96) as it will decode the entities for page/posts?

For example: ⬇️ 

```diff
diff --git a/packages/core-commands/src/site-editor-navigation-commands.js b/packages/core-commands/src/site-editor-navigation-commands.js
index ba34fffd988..1b6de1293c9 100644
--- a/packages/core-commands/src/site-editor-navigation-commands.js
+++ b/packages/core-commands/src/site-editor-navigation-commands.js
@@ -25,6 +25,7 @@ import { useDebounce } from '@wordpress/compose';
 import { useIsBlockBasedTheme } from './hooks';
 import { unlock } from './lock-unlock';
 import { orderEntityRecordsBySearch } from './utils/order-entity-records-by-search';
+import { decodeEntities } from '@wordpress/html-entities';
 
 const { useHistory } = unlock( routerPrivateApis );
 
@@ -93,7 +94,7 @@ const getNavigationCommandLoaderPerPostType = ( postType ) =>
 					name: postType + '-' + record.id,
 					searchLabel: record.title?.rendered + ' ' + record.id,
 					label: record.title?.rendered
-						? record.title?.rendered
+						? decodeEntities( record.title?.rendered )
 						: __( '(no title)' ),
 					icon: icons[ postType ],
 				};

```

Reply:
yes, that looks like a good place to me.

Reply:
Thanks, I have made the changes accordingly.